include/ogdf/basic/geometry.h

```diff
@@ -1054,4 +1055,38 @@ inline int orientation(const DSegment& s, const DPoint& p) {
 	return orientation(s.start(), s.end(), p);
 }
 
+/**
+ * Check whether this point lies within a node (using
+ * node shapes with same size and aspect as in TikZ).
+ *
+ * @param point the point
+ * @param v the node.
+ * @param vSize Width and height of v.
+ * @param shape of the node.
+ * @return true if the point lies within the border of \p v.
+ */
+OGDF_EXPORT bool isPointCoveredByNode(const DPoint& point, const DPoint& v, const DPoint& vSize,
+		const Shape& shape);
+
+/**
+ * returns the point where a vector, pointing from center in direction of angle, intersects with
+ * the countour of any regular n-polygon.
+ * \par angle angle of vector.
+ * \par n number of polygon sides.
+ * \par rotationOffset rotational offset of the polygon in RAD.
+ * \par center center point of the polygon.
+ * \par vSize Width and height of v.
+ */
+OGDF_EXPORT DPoint contourPointFromAngle(double angle, int n, double rotationOffset = 0,
+		const DPoint& center = DPoint(), const DPoint& vSize = DPoint(1, 1));
+/**
```

Comment:
I am using `util/test_number_newlines.sh -f` to check for such aesthetics, usually because there are too many new lines. The script should also check for cases like this one, where there is a line missing.